Libraries/Performance/NativeJSCSamplingProfiler.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import {Platform} from 'react-native';
+
+export interface Spec extends TurboModule {
+  +operationComplete: (token: number, result: ?string, error: ?string) => void;
+}
+
+export default (Platform.OS === 'android'
+  ? TurboModuleRegistry.getEnforcing<Spec>('JSCSamplingProfiler')
```

Comment:
Like in other PRs, this should call .get() instead, no need to check for platform